SpatialGDK/Extras/schema/virtual_worker_translation.schema

```diff
@@ -13,5 +13,5 @@ type VirtualWorkerMapping {
 
 component VirtualWorkerTranslation {
      id = 9979;
-     list<VirtualWorkerMapping> virtual_worker_mapping = 1;
+     transient list<VirtualWorkerMapping> virtual_worker_mapping = 1;
```

Comment:
With this change, the mapping data won't be put into any snapshots, although the entity will exist.